FsGrpc/FsGrpc.fsproj

```diff
@@ -23,9 +23,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="6.0.0" />
-    <PackageReference Include="Google.Protobuf" Version="3.19.4" />
-    <PackageReference Include="NodaTime" Version="3.0.9" />
+    <PackageReference Include="Google.Protobuf" Version="3.26.0" />

```

Comment:
did you mean to remove the FSharp.Core reference? I think this is needed to pin it to the lowest allowed version  for better compatibilty. Should probably be 8.0 now?